plugins/reach/lib/api/data/entryVendorTaks/KalturaEntryVendorTask.php

```diff
@@ -342,6 +343,23 @@ private function validateEntryId()
 			}
 		}
 	}
+
+	private function validateCatalogLimitations()
```

Comment:
Use Protected

Reply:
used private to be consistent with the other internal validate function